bundles/org.openhab.binding.icalendar/README.md

```diff
@@ -23,20 +25,54 @@ Each `calendar` thing requires the following configuration parameters:
 | `maxSize`           | The maximum size of the iCal-file in Mebibytes.                                                                                                                                           | mandatory (default available) |
 | `authorizationCode` | The authorization code to permit the execution of embedded command tags. If set, the binding checks that the authorization code in the command tag matches before executing any commands. | optional                      |
 
+Each `eventfilter` thing requires a bridge of type `calendar` and has following configuration options:
```

Comment:
And a a sub title ###Thing Configuration for "eventFilter" here

Reply:
OK.